pkg/controller/volume/attachdetach/util/util_test.go

```diff
@@ -283,7 +282,7 @@ func Test_CreateVolumeSpec(t *testing.T) {
 }
 
 func setup(nodeName string, t *testing.T) (*volume.VolumePluginMgr, csimigration.PluginManager, csitrans.CSITranslator, fakeframework.PersistentVolumeLister, fakeframework.PersistentVolumeClaimLister) {
-	tmpDir, err := utiltesting.MkTmpdir("csi-test")
+	tmpDir, err := os.MkdirTemp("", "csi-test")
```

Comment:
Should these `MkdirTemp(); if err, t.Fatal; defer os.RemoveAll` patterns be replaced with [`testing.T.TempDir`](https://pkg.go.dev/testing#T.TempDir)?

The five lines here can be replaced with one: `tmpDir := t.TempDir()`.

Reply:
Good catch, thank you! Let me fix that 🙂